Assets/GothicVR/Scripts/Caches/AssetCache.cs

```diff
@@ -161,7 +161,17 @@ public PxSoundData<float> TryGetSound(string key)
             if (soundCache.TryGetValue(preparedKey, out PxSoundData<float> data))
                 return data;
             
-            var wavFile = PxSound.GetSoundArrayFromVDF<float>(PhoenixBridge.VdfsPtr, $"{GetPreparedKey(key)}.wav");
+            PxSoundData<float> wavFile = new PxSoundData<float>();
+
+            try
+            {
+                wavFile = PxSound.GetSoundArrayFromVDF<float>(PhoenixBridge.VdfsPtr, $"{GetPreparedKey(key)}.wav");
```

Comment:
Do we need a try/catch? Why not returning a null? I feel PxCs should take care of it. Other phoenix-shared calls always return null if neccessary. So no "error" handling needed.